pyproject.toml

```diff
@@ -17,6 +17,9 @@ duckdb = "^0.7.1"
 [tool.poetry.group.dask.dependencies]
 dask = {extras = ["complete"], version = "^2023.3.1"}
 
+[tool.poetry.group.modin.dependencies]
```

Comment:
FYI: split the deps into groups to avoid install all deps for one test.
Issue : not able to install (looks like ray is not working on 3.11?)

Reply:
![Screenshot from 2023-03-15 20-15-14](https://user-images.githubusercontent.com/22365532/225419212-25266ab9-78e4-4355-84ac-9b9bed7739ab.png)
Strange, seems to work for me